source/tv/phantombot/cache/TwitchCache.java

```diff
@@ -459,7 +465,8 @@ public String getStreamStatus() {
     public void setStreamStatus(String streamTitle) {
         forcedStreamTitleUpdate = true;
         this.streamTitle = streamTitle;
-    }
+        EventBus.instance().postAsync(new TwitchTitleChangeEvent(streamTitle));  
+	}
```

Comment:
This brace looks out of place?  Should line up with the setStreamStatus() method?